### PR TITLE
misc: Italian language update

### PR DESCRIPTION
**Pull request by danielstc**
## What kind of change does this PR introduce?

- [x] Other, please describe:

Update of the Italian translation, following the introduction of the plugin system. Other minor corrections.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [ ] New/updated tests are included

### CODE REVIEW

File: src/renderer/i18n/locales/it-IT.js [hunk 1]
```diff
@@ -88,7 +89,7 @@ export default {
       TITLE: 'Il tuo Turno',
       KNOWLEDGE: 'Ora, armato della consapevolezza di quanto sia importante mantenere al sicuro le tue frasi segrete, sei pronto a rivendicare la tua autonomia finanziaria con ARK Desktop Wallet.',
       SUPPORT: 'ARK è un ecosistema open source e se hai bisogno di assistenza, la community e il team ARK sono qui per aiutarti. Crea un post su reddit.ark.io o unisciti al sistema di chat in tempo reale su slack.ark.io.',
-      CONCLUSION: 'Tutti qui, al team di ARK.io, speraiamo ti piacerà utilizzare l\'ARK Desktop Wallet e partecipare nella rivoluzione blockchain!'
+      CONCLUSION: 'Tutti qui, al team di ARK.io, speraiamo ti piacerà utilizzare l\'ARK Desktop Wallet e partecipare alla rivoluzione blockchain!'
```
Comment (dated) [inline src/renderer/i18n/locales/it-IT.js]:
```suggestion
      CONCLUSION: 'Tutti qui, al team di ARK.io, speriamo ti piacerà utilizzare l\'ARK Desktop Wallet e partecipare alla rivoluzione blockchain!'
```

File: src/renderer/i18n/locales/it-IT.js [hunk 2]
```diff
@@ -249,6 +254,14 @@ export default {
     TOOLTIP: 'La nuova versione ({version}) è stata rilasciata!'
   },
 
+  APP_SIDEMENU_PLUGIN_CONFIRMATION: {
+    QUESTION: 'Sei sicuro di voler abilitare il sistema di plugin?',
+    NO: 'No, tienilo disabilitato',
+    YES: 'Sì, abilitalo',
+    NOTICE: 'ATTENZIONE: questo sistema è attualmente in BETA. Tutti i plugin non ufficiali sono utilizzati a proprio rischio. Al momento non possiamo verificare la sicurezza di qualsiasi plug-in di terze parti e tutti i test sulla sicurezza e la scoperta delle vulnerabilità sono responsabilità dell\'autore del plug-in. Accettando questo avviso, l\'utente riconosce di utilizzare plugin di terze parti a proprio rischio.',
```
Comment (dated) [inline src/renderer/i18n/locales/it-IT.js]:
```suggestion
    NOTICE: 'ATTENZIONE: questo sistema è attualmente in BETA. Tutti i plugin non ufficiali sono utilizzati a proprio rischio. Al momento non possiamo verificare la sicurezza di qualsiasi plugin di terze parti e tutti i test sulla sicurezza e la scoperta delle vulnerabilità sono responsabilità dell\'autore del plugin. Accettando questo avviso, l\'utente riconosce di utilizzare plugin di terze parti a proprio rischio.',
```

File: src/renderer/i18n/locales/it-IT.js (hunk 1 above)
Comment (danielstc) [reply src/renderer/i18n/locales/it-IT.js]: Typo. Thank .

File: src/renderer/i18n/locales/it-IT.js (hunk 2 above)
Comment (danielstc) [reply src/renderer/i18n/locales/it-IT.js]: @dated
The word "_plugin_" also appears in other fields. At this point I'd change them all. Not just here. If you confirm I will proceed with the changes.

"plugin" ---> "plug-in"

Comment (danielstc) [reply src/renderer/i18n/locales/it-IT.js]: Don't worry about the comment above. Fixed. Ty

**Pull request merged**